docs/schemas/itemRaw.json

```diff
@@ -0,0 +1,733 @@
+        }
+      }
+    },
+
+    "tas-singleLineAnswer": {
+      "description": "Schema for singleLine answer",
+      "allOf": [
+         { "$ref": "#/definitions/tas-stringAnswer" },
+         { "properties": {
+             "type": {
+               "type": "string",
+               "enum": ["singleLine"]
+             }
+           }
+         }
+      ]
+    },
+
+    "tas-multilineAnswer": {
+      "description": "Schema for multiline answer",
+      "allOf": [
+         { "$ref": "#/definitions/tas-stringAnswer" },
+         { "properties": {
+             "type": {
+               "type": "string",
+               "enum": ["multiline"]
+             }
+           }
+         }
+      ]
+    },
+
+    "tas-numberAnswer": {
+      "description": "Schema for number answer",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["number"]
+        },
+        "value": {
+          "description": "Answer for this item",
+          "type": "number",
+          "multipleOf" : 1,
+          "minimum": -2147483648,
+          "maximum": 2147483647
+        }
+      }
+    },
+
+    "tas-boolAnswer": {
+      "description": "Schema for boolean answer",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["bool"]
+        },
+        "value": {
+          "description": "Answer for this item",
+          "type": "boolean"
+        }
+      }
+    },
+
+    "tas-resumeAnswer": {
+      "description": "Schema for resume answer",
+      "type": "object",
+      "allOf": [
+        { "$ref": "#/definitions/tas-file" },
+        { "properties": {
+            "type": {
+              "type": "string",
+              "enum": ["resume"]
+            },
+            "job": {
+              "description": "ID for the job associated with this item",
+              "type": "string"
+            },
+            "candidate": {
+              "description": "ID for the candidate associated with this item",
+              "type": "string"
+            },
+            "isTyped": {
+              "description": "Was the resume typed rather than sent as a file",
+              "type": "boolean",
+              "default": "false"
+            },
+            "typedString": {
+              "description": "The answer if the resume was typed rather than uploaded",
+              "type": "string"
+            }
+          }
+        }
+      ]
+    },
+
+    "tas-imageAnswer": {
+      "description": "Schema for image answer",
+      "type": "object",
+      "allOf": [
+        { "$ref": "#/definitions/tas-file" },
+        { "properties": {
+            "type": {
+              "type": "string",
+              "enum": ["image"]
+            },
+            "format": {
+              "description": "Mime type of file",
+              "type": "string"
+            }
+          }
+        }
+      ]
+    },
+
+    "tas-signatureAnswer": {
```

Comment:
I've removed it for now; we can always get it back from the commit history when we find a use for it.